packages/binutils.rb

```diff
@@ -23,7 +23,10 @@ class Binutils < Package
      x86_64: '784c5e9bfd884c411708ae3c2ee1c852022f411794ff109fb7d37e91f124037b'
   })
 
-  depends_on 'flex'
+  depends_on 'zlibpkg' # R
+  depends_on 'glibc' # R
+  depends_on 'elfutils' # R
+  depends_on 'flex' # R
```

Comment:
Since binutils is a core package, we'll need elfutils added to core_packages.txt.

Reply:
done